### DISCUSSION TITLE
TimeoutConfig default values

### DISCUSSION THREAD

**Discussion by ymwjbxxq** [smithy-lang/smithy-rs#1210] [Q&A]
Hello,

in node, I can see that many default values are 2 minutes
https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/Config.html#httpOptions-property

I am referring to httpOptions.timeout
 Sets the socket to timeout after timeout milliseconds of inactivity on the socket. Defaults to two minutes (120000).

I am unsure which is the equivalent in Rust, and I cannot find the default values.

For example:

```
    let config = aws_config::from_env()
        .retry_config(RetryConfig::new().with_max_attempts(10))
        .timeout_config(
            TimeoutConfig::new()
                .with_read_timeout(Some(Duration::from_secs(1)))
                .with_connect_timeout(Some(Duration::from_secs(1)))
                .with_api_call_timeout(Some(Duration::from_secs(1))),
        )
        .load()
        .await;
    let dynamodb_client = aws_sdk_dynamodb::Client::new(&config);
```

Could you please point me to the documentation or tell me the equivalent of httpOptions.timeout from the node? 

Thanks,
Dan

**Comment by jdisanti** [comment]
We haven't finished implementing timeouts yet, so currently they don't have any default values.
